### PR TITLE
New formula:  ImageUp: Display the "input source" anywhere

### PR DESCRIPTION
**Pull request by kkdd**

### CODE REVIEW

File: Casks/imageup.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+cask :v1 => 'imageup' do
+  homepage 'http://software.cockscomb.info/imageup/'
+  version '0.7.3'
+  url "http://download.cockscomb.info/software/imageup/ImageUp.zip"
```
Comment (vitorgalvao) [inline Casks/imageup.rb]: Double quotes are reserved for cases where string interpolation is needed. Please change these to single quotes.

File: Casks/imageup.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  sha256 'ff845d526f2e99e0802e529a5934907df58c02d2a0d8a24baabdc0ae2a7adc9e'
+  license :unknown
+  app 'ImageUp.app'
+  caveats <<-EOS.undent
```
Comment (vitorgalvao) [inline Casks/imageup.rb]: Remove the caveat. It is meant for installation instructions, not general info.

**Comment by vitorgalvao** [comment]
Seeing as `url` doesn’t point to a specific version, this cask should use `version :latest` and `sha256 :no_check`. [See an example](https://github.com/caskroom/homebrew-cask/blob/master/Casks/acorn.rb).

Please look at other casks and mimic their structure.

File: Casks/imageup.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+cask :v1 => 'imageup' do
+  homepage 'http://software.cockscomb.info/imageup/'
+  version '0.7.3'
+  url "http://download.cockscomb.info/software/imageup/ImageUp.zip"
+  sha256 'ff845d526f2e99e0802e529a5934907df58c02d2a0d8a24baabdc0ae2a7adc9e'
+  license :unknown
```
Comment (vitorgalvao) [inline Casks/imageup.rb]: `:gratis`

**Comment by kkdd** [comment]
Thank you. According to your advice, some modifications are introduced.

**Comment by Ngrd** [comment]
The stanza order should be completely different and also the license should be changed to `:gratis` as @vitorgalvao already mentioned earlier. The `name` stanza is also missing here and should be specified before the `homepage`.

In addition to, I would also add the appcast, even though there is no point in it at the moment, because it doesn't provide versioned download URL, but maybe in the feature the behaviour will change. The final variant should look like this:
```ruby
cask :v1 => 'imageup' do
  version :latest
  sha256 :no_check

  url 'http://download.cockscomb.info/software/imageup/ImageUp.zip'
  appcast 'http://updates.cockscomb.info/imageup/appcast.xml'
  name 'ImageUp'
  homepage 'http://software.cockscomb.info/imageup/'
  license :gratis

  app 'ImageUp.app'
end
```

**Comment by vitorgalvao** [comment]
Closing for lack of interest/reply from @kkdd.

Since the cask required a complete rewrite anyway, please feel free to submit it yourself if you so wish, @Ngrd.

**Pull request closed without merging**